SwiftGit2/References.swift

```diff
@@ -119,6 +120,27 @@ public struct Branch: ReferenceType {
 		}
 		commit = PointerTo<Commit>(oid)
 	}
+	
+	public func allCommits(in repo: Repository) -> [Commit] {
```

Comment:
Please add a comment explaining the return value of this method. For instance, are the returned commits sorted from newest to oldest?